src/App.js

```diff
@@ -0,0 +1,39 @@
+import {
+  Link,
+  Route,
+  Routes,
+  useLocation,
+} from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faChevronLeft, faMicrophone, faGear } from '@fortawesome/free-solid-svg-icons';
+import './App.css';
+import PollutionApp from './components/pollutionApp';
+import SingleCity from './components/singleCity';
+
```

Comment:
- You have done great building out your capstone :+1: But there are a few changes and requirements you need to complete before moving to the next milestone. :smile: 

> It's a list of metrics that can be filtered by the country (imagine a search field to introduce the country name like Italy, Croatia, etc.). This page should be your homepage.

- [ ] As the previous reviewer suggested, and also the requirement says you should have a search or a filter of some sort in your home page that a user can use to search or filter through the teams in your Home page.  You find a way to group the city or you can make users search by names of the CAPITAL CITY :+1: :+1: :smile:

Reply:
This is what the requirements say : 

When the page loads, the webapp shows the list of categories that could be filtered out by some parameter, for example by the category name.

Please notice that the requirements say "BY EXAMPLE THE CATEGORY NAME". It is an example. I chose another filter. You can see it on the dropdown menu located below the navbar.